webrtc.html

```diff
@@ -4978,7 +4979,7 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
             "idlMemberType"><a>DOMString</a></span></dt>
             <dd>
               <p>The codec MIME type. Valid types are listed in
-              [[IANA-RTP-2]].</p>
+              [[IANA-RTP-2]]. RTP retransmission MIME types, as defined in [[!RFC4588]], are not allowed.</p>
```

Comment:
I think this restriction is at the wrong level. The RTCRtpCodecParameters dictionary may turn out to be useful somewhere else; it's the "codecs" attribute that needs to have the restriction.